test/vstest.console.UnitTests/ExecutorUnitTests.cs

```diff
@@ -60,25 +60,17 @@ public void ExecutorPrintsSplashScreenTest()
 
 
         /// <summary>
-        /// Executor should try find "project.json" if empty args given
+        /// Executor should Print Error message and Help contents when no arguments are provided.
         /// </summary>
         [TestMethod]
-        public void ExecutorEmptyArgsCallRunTestsProcessor()
+        public void ExecutorEmptyArgsPrintsErrorAndHelpMessage()
         {
             var mockOutput = new MockOutput();
             var exitCode = new Executor(mockOutput, this.mockTestPlatformEventSource.Object).Execute(null);
 
-            // Since no projectjsons exist in current folder it should fail
-            Assert.AreEqual(1, exitCode, "Exit code must be One for bad arguments");
-
-            //// Verify that messages exist
-            //Assert.IsTrue(mockOutput.Messages.Count > 0, "Executor must print atleast copyright info");
-            //Assert.IsNotNull(mockOutput.Messages.First().Message, "First Printed Message cannot be null or empty");
+            Assert.AreEqual(1, exitCode, "Exit code must be One when no arguments are provided.");
```

Comment:
We should also check that help content is getting shown on screen. Its seems like there is no way here to figure this out. 
I would suggest to add acceptance test for it.